requirements.txt

```diff
@@ -0,0 +1,4 @@
+matplotlib
```

Comment:
This `requirements.txt` files is for creating PyPI releases. In order to automate PyPI releases whenever we have a tagged release, you may want to add a pypi.yaml file similar to [GeoCAT-comp's PyPI YAML file](https://github.com/NCAR/geocat-comp/blob/main/.github/workflows/pypi.yaml) unless you have plans of manually releasing PyPI versions.

Reply:
Oh, neat, I've never run the pypi releases. I added a similar file, but I think I need to add PYPI_PASSWORD as a secret, like in comp. How do I do that?

Reply:
I have created and added the secret "PYPI_GEOCAT_VIZ" to this repo for you. You should just make sure to refer to exact that name within your pypi.yaml file.

FYI: The steps for PyPI configuration is as follows:

1. Create an account on pypi.org
2. Create an API token from https://pypi.org/manage/account/ that will be used to upload the built package. 
3. Add the API token as a secret to your GitHub repo settings. The secret's name must be referred to from with in `pypi.yaml` file in your repo.